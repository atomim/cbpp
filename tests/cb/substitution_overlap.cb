#define AB "oy"
#define ABBA "waterloo"
#define SUM(a,b) (a+b)
#define SQUARESUM(a,b) (a*a+b*b)

AddText "abba + ab + abba = " + ABBA + AB + ABBA
AddText "result: " + SQUARESUM(2,3)
DrawScreen
WaitKey