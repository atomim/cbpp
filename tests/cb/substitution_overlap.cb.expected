// #define AB "oy"
// #define ABBA "waterloo"
// #define SUM(a,b) (a+b)
// #define SQUARESUM(a,b) (a*a+b*b)

AddText "abba + ab + abba = " + "waterloo" + "oy" + "waterloo"
AddText "result: " + (2*2+3*3)
DrawScreen
WaitKey